build.gradle

```diff
@@ -94,7 +95,9 @@ compileTestJava {
 }
 
 dependencies {
-    compile 'org.broadinstitute:barclay:1.0.0'
+    compile 'org.broadinstitute:barclay:1.0.0-2-gb718440-SNAPSHOT'
+    // javadoc utilities
+    compile(files(((URLClassLoader) ToolProvider.getSystemToolClassLoader()).getURLs()))
```

Comment:
Yes, thats a good idea. I realized today that now its including tools.jar in the uber jar, which we can't have and don't want.